src/index.ts

```diff
@@ -29,7 +30,7 @@ app.set('view engine', 'hbs')
 app.use((req: Request, res: Response, next: NextFunction) => {
   res.locals.projectName = config.projectName
   res.locals.baseUrl = config.baseUrl
-  res.locals.pathPrefix = config.baseUrl ? '' : '/'
+  res.locals.pathPrefix = path.join(config.baseUrl, './')
```

Comment:
This does not work @aeneasr and it is not self explanatory at all. What's the reason for this change?

```
$ node
require("path")
path.join("https://website.com/profile", "./")
```

produces

```
'https:/website.com/profile/'
```

verbatim. So this is terribly incorrect.